src/main/include/subsystems/Climber.h

```diff
@@ -0,0 +1,25 @@
+#include <rev/SparkMax.h>
+#include <frc2/command/SubsystemBase.h>
+#include <frc/controller/PIDController.h>
+#include "Constants.h"
+#include <frc2/command/CommandPtr.h>
+
+using namespace rev::spark;
+
+class Climber : public frc2::SubsystemBase {
+    public :
+
+    Climber();
+
+    frc2::CommandPtr FlipArm();
```

Comment:
It looks as if the convention that FIRST uses is to add `Command` to the end of any method that returns a command (i.e., `frc2::CommandPtr`). See the [`Intake` subsystem in their `RapidReactCommandBot` example](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibcExamples/src/main/cpp/examples/RapidReactCommandBot/cpp/subsystems/Intake.cpp).

Therefore, suggest that we follow their lead:

```suggestion
    frc2::CommandPtr FlipArmCommand();
```

Reply:
Yes. If it is a Command, it should called as such.

Reply:
changed name to be a command